Add r-suppdists

qwhelan opened the pull request:
Add the ```SuppDists``` R package from CRAN.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/r-suppdists/recipe```) and found it was in an excellent condition.

bgruening commented on recipes/r-suppdists/recipe/meta.yaml:
```diff
@@ -0,0 +1,46 @@
+    - m2w64-toolchain  # [win64]
+    - gcc  # [not win]
+  run:
+    - r-base
```
please add libgcc not not-win.

qwhelan commented:
@bgruening Apologies for the delay - was on vacation.

Incorporated your ```libgcc``` change, added you and @johanneskoester as maintainers, and rebased.

bgruening commented:
LGTM. ping @isuruf :)

isuruf commented on recipes/r-suppdists/recipe/meta.yaml:
```diff
@@ -0,0 +1,48 @@
+    - r-base
+    - posix  # [win64]
+    - m2w64-toolchain  # [win64]
+    - libgcc  # [not win64]
```
This should be moved to `run` and 
```
- gcc  # [not win]
```
added to build

qwhelan commented:
@isuruf  Done.
